lib/splitclient-rb/validators.rb

```diff
@@ -38,6 +41,25 @@ def valid_matcher_arguments(args)
       true
     end
 
+    def valid_flag_sets(method, flag_sets)
+      if flag_sets.nil? || !flag_sets.is_a?(Array)
+        return nil
+      end
+      valid_flag_sets = SortedSet[]
+      flag_sets.compact.uniq.select do |flag_set|
```

Comment:
you are not trimming the `flag set` if neecessary.

Reply:
The trimming is done inside the loop, I need to check first if the object is string, otherwise the strip method will raise exception